vendor/github.com/opencontainers/image-spec/specs-go/descriptor.go

```diff
@@ -1,27 +0,0 @@
-// Copyright 2016 The Linux Foundation
```

Comment:
I think it's because your run `make update-deps` and forget to `git add vendor/github.com/opencontainers/image-spec/specs-go/v1/descriptor`, 
`descriptor.go` was in `vendor/github.com/opencontainers/image-spec/specs-go` before bumping.